CHANGELOG.md

```diff
@@ -36,6 +36,13 @@
    * Applications getting ref's to `HoistInputs` should be aware that these ref's now return a ref to
    a `HoistInputModel`.   In order to get the dom element associated with the component use the new
    `domRef` property of that model rather than `HoistComponent.getDOMNode()` method.
+* Hoist-React grids now require ag-Grid v24.1.0 or higher - update your ag-Grid depency in your app's `package.json` file.  
+  ag-Grid v24.1.0 [lists the following breaking changes](https://www.ag-grid.com/ag-grid-changelog/):
```

Comment:
would simplify this to just mention the ones we *are* concerned about

Reply:
Done.